initial-promises/node-server/failsafe.cf

```diff
@@ -192,3 +243,10 @@ body file_select exclude(name)
           file_result => "!leaf_name";
 }
 
+body file_select by_fullname(names)
+# @brief Select files that match `names`
+# @param names A regular expression
```

Comment:
Once in our standard library that's included in a standard run and once in failsafe.cf because failsafe.cf doesn't use the standard libraries, only the ones in failsafe.cf itself - this is to avoid a broken stdlib breaking failsafe.cf.